hummingbot/connector/derivative/binance_perpetual/binance_perpetual_derivative.py

```diff
@@ -680,10 +688,10 @@ async def _status_polling_loop(self):
                 await self._poll_notifier.wait()
                 await safe_gather(
                     self._update_balances(),
-                    self._update_positions(),
-                    self._update_order_fills_from_trades(),
-                    self._update_order_status()
+                    self._update_positions()
                 )
+                await self._update_order_fills_from_trades(),
+                await self._update_order_status()
```

Comment:
Why did you move out of the safe_gather these 2 methods? Shouldn't they be enforced to be grouped together in the same tick with the gather?

Reply:
I don't want those 2 tasks to run concurrently. It'll prevent cases where orders complete event comes in without a filled event when api backup kick in.

Reply:
Both tasks shouldn't ideally run concurrently.